index.css

```diff
@@ -79,4 +79,20 @@
     .font-bold {
         @apply antialiased;
     }
+
+    .debug, .debug-red {
```

Comment:
@nemanjakrstic @manojx031 I use these classes a lot in my personal life. I left this here to help us. What do you think? 

My initial goal is to throw a lint error on this or something to prevent this from going into prod